apps/root/src/services/meanApiService.ts

```diff
@@ -42,14 +44,22 @@ export default class MeanApiService {
 
   providerService: ProviderService;
 
+  accountService: AccountService;
+
   client: ethers.providers.Web3Provider;
 
   loadedAsSafeApp: boolean;
 
-  constructor(contractService: ContractService, axiosClient: AxiosInstance, providerService: ProviderService) {
+  constructor(
+    contractService: ContractService,
+    axiosClient: AxiosInstance,
+    providerService: ProviderService,
+    accountService: AccountService
+  ) {
     this.axiosClient = axiosClient;
     this.contractService = contractService;
     this.providerService = providerService;
+    this.accountService = accountService;
```

Comment:
With the implementation of `accountService`, what are your thoughts on removing the `accountId` parameter from the six methods associated with labels and the contact list?

Reply:
Good thinking! Will implement